frontend/src/App.js

```diff
@@ -44,6 +45,45 @@ class App extends Component {
         this.listener();
     }
 
+    // const [uid, setUid] = useState("");
+    // const [isAuthenticated, setAuthStatus] = useState(false);
+
+    // useEffect(() => {
+    //     auth.onAuthStateChanged((data) => {
+    //         if (data) { //logged in
+    //             setUid(data.uid);
+    //             setAuthStatus(true);
+    //             console.log("true: " + isAuthenticated);
+    //         }
+    //         else {
+    //             setUid("");
+    //             setAuthStatus(false);
+    //             console.log("false: " + isAuthenticated);
+    //         }
+    //     });
+    // }, []);
+
+    signOut = () => {
```

Comment:
is this a new function? Don't we have this in Navbar.js?

Reply:
I think this is from haiwei's old code when I merged her branch to implement redux. I'll take it out!